weather_data_analyzer.py

```diff
@@ -0,0 +1,198 @@
+import csv
+import sys
+from weather_record import WeatherRecord
+from report_types \
+    import AverageTempHumidReportType, HighLowTempHumidityReportType
+
+dummy_record_for_comparison = {
+    'PKT': "2022-08-16",
+    'Mean TemperatureC': None,
+    'Max TemperatureC': -sys.maxsize,
+    'Min TemperatureC': sys.maxsize,
+    ' Min Humidity': sys.maxsize,
+    'Max Humidity': -sys.maxsize,
+    ' Mean Humidity': None
```

Comment:
```suggestion
    'PKT': "2022-08-16",
    'Mean TemperatureC': None,
    'Max TemperatureC': -sys.maxsize,
    'Min TemperatureC': sys.maxsize,
    'Min Humidity': sys.maxsize,
    'Max Humidity': -sys.maxsize,
    'Mean Humidity': None,
```

Reply:
We can't do this. Because the fields are set according to the files header. The files has header `' Min TemperatureC'`, `' Mean Humidity'`.

One way is to correct the headers of all files first but If i correct the header of files on my laptop, then it will not work for you or any other user.